cluster-config/staging/stone-stg-m01/kustomization.yaml

```diff
@@ -1,17 +1,23 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../member-base
+- ../member-base
 patches:
-  - path: tekton-results-database-secret-path.yaml
```

Comment:
it's very difficult to review when the indentation changes. Can you please revert the indentation change?